<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depos_window_2</class>
 <widget class="QMainWindow" name="depos_window_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Депозитный калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_summ">
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <double>100000000000000007629769841091887003294964970946560.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>1000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_summ">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Сумма вклада</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spinBox_day">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="value">
           <number>120</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_day">
          <property name="text">
           <string>Срок размещения</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_period">
          <item>
           <property name="text">
            <string>В днях</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>В месяцах</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>В годах</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QDoubleSpinBox" name="proc_doubleSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_proc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Процентная ставка депозита</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QDoubleSpinBox" name="nalog_doubleSpinBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_nalog">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Налоговая ставка</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="capit_checkBox">
          <property name="text">
           <string>Капитализация</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Переодичность выплат</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Каждый день</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Каждую неделю</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Каждый месяц</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Каждый квартал</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Каждый год</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>В конце периода</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Тип
операции</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Дата
операции</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Сумма
операции</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Удалить
операцию</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="calc_pushButton_2">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDateEdit" name="start_dateEdit">
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Дополнительные вложения или снятия</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="start_date_lable">
        <property name="text">
         <string>Дата начала вклада</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>495</width>
         <height>83</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calc_pushButton">
      <property name="text">
       <string>Расчитать</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Сумма счета на конец периода</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_summ_all_out">
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="rightMargin">
         <number>40</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Налог</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_nalog_out">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Сумма процентов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_summ_out">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Тип</string>
    </property>
    <addaction name="main_calc_buttom"/>
    <addaction name="Credit_calc_button"/>
    <addaction name="depos_calc_button"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="main_calc_buttom">
   <property name="text">
    <string>Математический</string>
   </property>
  </action>
  <action name="Credit_calc_button">
   <property name="text">
    <string>Кредитный</string>
   </property>
  </action>
  <action name="depos_calc_button">
   <property name="text">
    <string>Депозитный</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
